styles.scss

```diff
@@ -248,10 +245,10 @@ a:link {
 }
 
 .image-blog__article {
-  padding: .62rem;
+  padding: .5rem;
   width: 24rem;
```

Comment:
Is this something you want to change size if the overall font size of the web page changes?